components/org.wso2.carbon.identity.sso.agent/src/main/java/org/wso2/carbon/identity/sso/agent/SSOAgentConstants.java

```diff
@@ -70,6 +70,8 @@ private SSOAgentConstants() {
         public static final String OAUTH2_SAML2_GRANT_URL = "OAuth2SAML2GrantURL";
         public static final String SKIP_URIS = "SkipURIs";
         public static final String QUERY_PARAMS = "QueryParams";
+        public static final String Secretkey_FilePath = "/home/nilasini/Documents/tomcat/apache-tomcat-8.0.45/webapps/travelocity" +
```

Comment:
Are you trying to ship your laptop with this code :stuck_out_tongue_closed_eyes: Or asking the user to make sure always install tomcat in "/home/nilasini/Documents/tomcat/apache-tomcat-8.0.45/". Nice way to put your name in the code :stuck_out_tongue_closed_eyes: Please find a way to get the path. And my preference is to put the file in little higher level. Like in webapps folder.